os/kernel/binary_manager/Kconfig

```diff
@@ -3,6 +3,12 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
+config USE_BP
```

Comment:
This new config covers too many parts, bm, bootloader and bootparam. I don't know this is correct. 
1. If we keep only this config, then this should be located at outside of bm.
2. We could add two more configs for bootloader. Current one covers bm and bp. And then make dependencies.
 i.e.
   in rtl8721csm kconfig
   config bootloader_fixed_partitions
       depends on !use_bp
   config bootloader_dynamic_partitions
         depends on use_bp
I suggest this because bootloader and others are different topics and modules.

Reply:
Done added followed option 2, added 2 additional configs for bl.